cluster/vagrant/provision-utils.sh

```diff
@@ -179,6 +180,7 @@ apiVersion: v1
 kind: Config
 clusters:
 - cluster:
+    server: "https://${MASTER_IP}"
     insecure-skip-tls-verify: true
```

Comment:
seeing this for a remote connection makes me really unhappy, needs an open followup issue